src/main/java/org/folio/service/inventory/InventoryHoldingManager.java

```diff
@@ -317,4 +320,43 @@ public Future<String> handleHoldingsRecord(CompositePoLine compPOL, Location loc
     }
   }
 
+  public Future<List<JsonObject>> getHoldingsForAllTenants(CompositePoLine poLine, RequestContext requestContext) {
+    var futures = getHoldingsFutures(poLine, requestContext);
+    return GenericCompositeFuture.all(new ArrayList<>(futures.values()))
+      .map(ar -> futures.values().stream()
+        .flatMap(future -> future.result().stream())
+        .toList()
+      );
```

Comment:
futures is sound strange, from reader prespective it is being hard to understand what is futures and future. Better to give better name or we can directrly return List in getHoldingsFutures() method instead of creating new ArrayList<> in each operation

Reply:
`futures` is a map of features to get holdings by `tenantId`: `Map<String, Future<List<JsonObject>>>`
Complication is that only list is not enough: we need to know for which tenant we fetched particular `List<JsonObject>`. So what's your idea?

Reply:
I thought it can be return just List, since Map structure important that make sense then. just better name futures varibales to make it more self-explanatory